frontend/packages/vkt/src/components/publicExamEvent/PublicExamEventGrid.tsx

```diff
@@ -0,0 +1,59 @@
+import { Alert, Grid, Paper } from '@mui/material';
+import { Trans } from 'react-i18next';
+import { H1, H2, HeaderSeparator, Text } from 'shared/components';
+import { APIResponseStatus, Severity } from 'shared/enums';
+
+import { PublicExamEventListing } from 'components/publicExamEvent/listing/PublicExamEventListing';
+import { PublicExamEventGridSkeleton } from 'components/skeletons/PublicExamEventGridSkeleton';
+import { useAppTranslation } from 'configs/i18n';
+import { useAppSelector } from 'configs/redux';
+import { publicExamEventsSelector } from 'redux/selectors/publicExamEvent';
+
+export const PublicExamEventGrid = () => {
+  // I18
+  const { t } = useAppTranslation({ keyPrefix: 'vkt.pages.homepage' });
+  // Redux
+  const { status, publicExamEvents } = useAppSelector(publicExamEventsSelector);
+
+  // State
+  const hasResults = publicExamEvents.length > 0;
+  const isLoading = status === APIResponseStatus.InProgress;
+
+  return (
+    <>
+      <Grid item className="public-homepage__grid-container__item-header">
+        <H1 data-testid="public-homepage__title-heading">{t('title')}</H1>
+        <HeaderSeparator />
+        <Text>{t('description')}</Text>
+      </Grid>
+      <Grid item className="public-homepage__grid-container__item-exam-dates">
+        <Paper elevation={3} className="public-homepage__exam-dates">
+          {isLoading ? (
+            <PublicExamEventGridSkeleton />
+          ) : (
+            <>
+              <Alert
+                className="public-homepage__exam-dates__heading-description"
+                severity={Severity.Info}
+              >
+                {t('note')}
+              </Alert>
+              <H2 className="public-homepage__exam-dates__heading-title">
+                {t('examinationDates.title')}
+              </H2>
+              <Text>
+                <Trans t={t} i18nKey="examinationDates.description"></Trans>
```

Comment:
Poistetaan Trans-elementti jos tuolle ei jää tänne tarvetta

Reply:
Näköjään toi Trans-komponentti tukee markuppia käännöstekstien sisällä, minkä takia sitä varmaan on tässä käytetty. Noi strong-tagit hajoaa jos antaa sellaisenaan käännösfunktiolle niin kuin yleensä on tehty. Sinänsä tietysti vähän poikkeus muuhun codebaseen, mutta ehkä tuon voisi antaa olla.

Reply:
Hyvä huomio. Musta ihan ok jättää näin, voitanee jatkossakin samaa konventiota käyttää jos tarve lisäillä esim. strong- tai i-tageja lokalisaation sisään.